tests/test_data.py

```diff
@@ -114,22 +114,6 @@
     "Success": "1 annotations"
 }
 
-alert_D = {
```

Comment:
this is not actually removed. github is messing up the diffs

Reply:
This got moved down, so the diff is valid, but I verified that there is no change so we are good.